boards/common/adafruit-nrf52-bootloader/reset.c

```diff
@@ -7,7 +7,7 @@
  */
 
 /**
- * @ingroup     boards_feather-nrf52840
+ * @ingroup     boards_common_adafruit-nrf52840-bootloader
```

Comment:
```suggestion
 * @ingroup     boards_common_adafruit-nrf-bootloader
```

probably in other places, too

Reply:
That seems to be the only occurrance:
```
RIOT/boards$ grep -Rni . -e adafruit-nrf52840-bootloader
./common/adafruit-nrf52-bootloader/reset.c:10: * @ingroup     boards_common_adafruit-nrf52840-bootloader
```